db/1.14/modules/ROOT/pages/database-connector-input-output-parameters.adoc

```diff
@@ -0,0 +1,219 @@
+. Drag the *Stored procedure* operation to your canvas that retrieves the order information.
+. Select the operation from your flow.
+. In the *SQL Query text* field, enter `{call UpdateStockLevel(:productId, :quantity, :updatedStockLevel)}`.
++
+This defines the SQL statement for calling a stored procedure UpdateStockLevel with input parameters productId and quantity, and an output parameter updatedStockLevel.
+. Set *In out parameters* to *Edit inline*.
+. Click the plus icon to open the *In out parameter* window.
+. In the *In out parameter* window, set the following parameters:
++
+[%header%autowidth.spread]
+|===
+|Key | Value
+|productId | #[payload.productId]
+|quantity | #[payload.quantity]
+|updatedStockLevel | (empty)
+|===
++
+[start=8]
+. Click *Finish*.
+
+The configuration enables you to execute the stored procedure, providing input parameters product ID and quantity, and retrieving the updated stock level as an output parameter.
+
+In the *Configuration XML* editor, the configuration looks like this:
+[source,xml,linenums]
+----
+<db:stored-procedure doc:name="Stored procedure"			
+<db:sql ><db:in-out-parameters >
+			</db:in-out-parameters><![CDATA[{call UpdateStockLevel(:productId, :quantity, :updatedStockLevel)}]]>
+			</db:sql>
+			<db:in-out-parameters >
+				<db:in-out-parameter key="updatedStockLevel" />
+				<db:in-out-parameter key="productId" value="#[payload.productId]" />
+				<db:in-out-parameter key="quantity" value="#[payload.quantity]" />
+			</db:in-out-parameters>
+----
+
+== Configure Custom Types
+
+The ability to use custom types as parameters in stored procedures depends on the database system you are using and the capabilities of the Database Connector or framework you are working with. 
+Exercise caution when you work with an Oracle database vendor and use the *Stored procedure* operation with custom database types. 
+When you define the stored procedure within a package, set the column type of a custom type parameter in the connector connection settings:
+
+. In Studio, open the Database Config Global Element Properties window.
+. Click the *Advanced* tab.
+. Set *Column types* to *Edit inline*.
+. Click the plus sign to open the *Column types* window.
+. In the *Column types* window, set *Id* and *Type name* fields with your desired values.
+. Click *OK*.
+
+image::database-column-type.png[Column type configuration window]
+
+In the *Configuration XML* editor window, the configuration looks like this:
+
+[source,xml,linenums]
+----
+<db:column-types >
+<db:column-type id="2003" typeName="MY_CUSTOM_PKG.MY_CUSTOM_TYPE" />
+</db:column-types>
+----
+
+It remains a good practice to define the *Column types* of a custom type parameter, even if the stored procedure is not part of a package but is directly defined.
+
+== Configuration Limitations
+Consider the following limitations when you configure *In out parameters*.
+
+=== Variable Values Reflection
```

Comment:
Is this really "Reflection"? Maybe you can think of another header here. Maybe "Variable Values in Stored Procedures."